test/e2e/apimachinery/custom_resource_definition.go

```diff
@@ -42,29 +51,152 @@ var _ = SIGDescribe("CustomResourceDefinition resources", func() {
 		framework.ConformanceIt("creating/deleting custom resource definition objects works ", func() {
 
 			config, err := framework.LoadConfig()
-			if err != nil {
-				e2elog.Failf("failed to load config: %v", err)
-			}
-
+			framework.ExpectNoError(err, "loading config")
 			apiExtensionClient, err := clientset.NewForConfig(config)
-			if err != nil {
-				e2elog.Failf("failed to initialize apiExtensionClient: %v", err)
-			}
+			framework.ExpectNoError(err, "initializing apiExtensionClient")
 
 			randomDefinition := fixtures.NewRandomNameCustomResourceDefinition(v1beta1.ClusterScoped)
 
-			//create CRD and waits for the resource to be recognized and available.
+			// Create CRD and waits for the resource to be recognized and available.
 			randomDefinition, err = fixtures.CreateNewCustomResourceDefinition(randomDefinition, apiExtensionClient, f.DynamicClient)
-			if err != nil {
-				e2elog.Failf("failed to create CustomResourceDefinition: %v", err)
-			}
+			framework.ExpectNoError(err, "creating CustomResourceDefinition")
 
 			defer func() {
 				err = fixtures.DeleteCustomResourceDefinition(randomDefinition, apiExtensionClient)
-				if err != nil {
-					e2elog.Failf("failed to delete CustomResourceDefinition: %v", err)
+				framework.ExpectNoError(err, "deleting CustomResourceDefinition")
+			}()
+		})
+
+		/*
+			Release : v1.16
+			Testname: Custom Resource Definition, list
+			Description: Create a API extension client, define 10 random custom resource definitions and list them using a label selector. API server MUST be able to list the custom resource definitions and delete them via delete collection.
+		*/
+		framework.ConformanceIt("listing custom resource definition objects works ", func() {
```

Comment:
this promotes a test to conformance without adding it to:
https://github.com/kubernetes/kubernetes/blob/master/test/conformance/testdata/conformance.txt

the process should be: add a test, then after it matches the conformance requirements, promote it to `ConformanceIt`.

/assign @spiffxp

Reply:
Fixed